lua/entities/base_wire_entity.lua

```diff
@@ -11,11 +11,14 @@ ENT.IsWire = true
 
 if CLIENT then
 	local wire_drawoutline = CreateClientConVar("wire_drawoutline", 1, true, false)
-	local beingLookedAtByLocalPlayer
+	-- Funny reverse-detour because this function is sometimes nil and sometimes not, but is never nil when drawing for the first time.
+	local function beingLookedAtByLocalPlayer(self)
+		beingLookedAtByLocalPlayer = BaseClass.BeingLookedAtByLocalPlayer
```

Comment:
This needs to be
```lua
local beingLookedAtByLocalPlayer
function beingLookedAtByLocalPlayer(self)
    beingLookedAtByLocalPlayer = BaseClass.BeingLookedAtByLocalPlayer
    return beingLookedAtByLocalPlayer(self)
end
```

Reply:
I know that functions inside a table can't access the table local unless this is done, but maybe this does work for local functions.

Reply:
According to the Lua standard
```lua
local function func() ... end
```
is syntactic sugar for
```
local func
func = function() ... end
```

Reply:
That's good. Wish it did the same thing for tables.